extensions_admin/pulp_rpm/extensions/admin/repo_list.py

```diff
@@ -34,8 +34,10 @@ def get_repositories(self, query_params, **kwargs):
             # sure it's there before proceeding
             if 'importers' not in r:
```

Comment:
as another option you could just check that 'importers' is not empty list:
`if 'importers' not in r or r['importers'] == []:`